modules/mri_violations/test/TestPlan.md

```diff
@@ -1,47 +1,64 @@
 MRI Violations Module Test Plan:
 
-1.  Check user access. Permission – `Violated Scans: View all-sites Violated Scans`
+1.  Check user access. Permission – `Violated Scans: View all-sites Violated 
+    Scans`
     [Automation Testing]
 2.  Check Selection Filter results for each of Patient Name, File Name,
     Series Description or Scan Type, Time Run, Type of Problem, SeriesUID
     [Manual Testing]
-3.  Filter for scans that violate MRI Protocol violation (For each type of Problem : Could not identify scan type, Visit Label does not exist, MRI Protocol Check Violation, Candidate mismatch)
-    Ensure link under Problem column goes to MRI Protocol Violations page with correct scan information 
-    set in the filters, and parameters showing when could not identify scan type, or MRI Protocol Check Violation are the type of problem
+3.  Filter for scans that violate MRI Protocol violation (For each type of 
+    Problem : 'Could not identify scan type', 'Visit Label does not exist',  
+    'Protocol Violation', 'CandID and PSCID doe not match' and 'CandID 
```

Comment:
`do` not `doe`:)

Reply:
@MounaSafiHarab Thanks! Just corrected the typo.